cfme/tests/control/test_default_alerts.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+
+import os
+import pytest
+import yaml
+
+from cfme.utils.path import data_path
+
+@pytest.fixture()
+def default_alerts(appliance):
+    file_name = data_path.join('ui/control/default_alerts.yaml').strpath
+    alerts = {}
+    if os.path.exists(file_name):
+        with open(file_name,'r') as f:
+            all_alerts = yaml.load(f)
+            if '5.9' in appliance.version.vstring:
```

Comment:
This can be simplified to ```if appliance.version == "5.9"```

Reply:
Won't this fail due to the z-stream numbers? e.g. 5.9.5.3

Reply:
Led you down the wrong path here, look into `VersionPicker()` here.

Reply:
changed to VersionPicker()